Source/JavaScriptCore/inspector/protocol/CSS.json

```diff
@@ -268,6 +268,21 @@
                 { "name": "synthesizedOblique", "type": "boolean", "optional": true, "description": "Whether the font has synthesized its obliqueness or not" }
             ]
         },
+        {
+          "id": "FontFace",
+          "type": "object",
+          "description": "A FontFace",
+          "properties": [
+              { "name": "family", "type": "string", "description": "Font Family Name"},
+              { "name": "weight", "type": "string", "description": "Font Stretch"},
+              { "name": "stretch", "type": "string", "description": "Font Family Name"},
+              { "name": "unicodeRange", "type": "string", "description": "Font Unicode Range"},
+              { "name": "featureSettings", "type": "string", "description": "Font Feature Settings"},
+              { "name": "display", "type": "string", "description": "Font Display"},
+              { "name": "sizeAdjust", "type": "string", "description": "Font Size Adjust"},
+              { "name": "loaded", "type": "string", "description": "Font Loaded"}
```

Comment:
is this actually a useful piece of information for the developer?

are there any actions they can take given this info?  or is it purely informative?

if the latter, id suggest not including it as Web Inspector usually tries to only provide information that is actionable, as otherwise it's potentially confusing